### ISSUE TITLE
Could not find :unspecified:

### ISSUE THREAD

**Issue by nikul014** [opentok/opentok-android-sdk-samples#386]
Hi,

I was trying to upgrade the opentok from version **2.19.1** to **2.20.0** and it throws the below error in all the projects. 

Could not determine the dependencies of task ':modules:compileDebugKotlin'.
> Could not resolve all files for configuration ':modules:debugRuntimeClasspath'.
   > Could not find :unspecified:.
     Searched in the following locations:
       - file:StudioProjects/testing/modules/libs/unspecified.jar
     Required by:
         project :modules > project :tele > com.opentok.android:opentok-android-sdk:2.20.0

How can I fix this issue in version 2.20.0?

Thanks

**Comment by Iannnr** [comment]
Same here, seems to be an issue with 2.20 and the maven repository. Even following the reference from https://tokbox.com/developer/sdks/android/ 
```
allprojects {
    repositories {
        ...
        mavenCentral()
        ...
    }
}
```
then 
```
dependencies {
    ...
    implementation 'com.opentok.android:opentok-android-sdk:2.20.0'
}
```
still isn't resolved on project build

**Comment by adrienrx** [comment]
Have you removed maven { url "http://tokbox.bintray.com/maven" } ?

**Comment by nikul014** [comment]
Fixed in the version 2.20.1 version.

**Comment by nikul014** [comment]
Fixed in the version 2.20.1 version.
